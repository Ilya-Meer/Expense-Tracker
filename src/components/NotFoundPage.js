import React, { Component } from 'react';
import ReactDOM from 'react-dom';


const NotFoundPage = () => (
  <div>
    <p>404!</p>
    <Link to="/">Home</Link>  
  </div>
)

export default NotFoundPage;