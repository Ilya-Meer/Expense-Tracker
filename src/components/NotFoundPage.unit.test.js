import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from './NotFoundPage';


describe('Not Found Page', () => {
  it('should render the Not Found Page', () => {
    const wrapper = shallow(<NotFoundPage />);
    expect(wrapper).toMatchSnapshot();
  })
})
