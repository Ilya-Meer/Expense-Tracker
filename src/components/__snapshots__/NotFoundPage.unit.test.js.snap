// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not Found Page should render the Not Found Page 1`] = `
<div>
  <p>
    404!
  </p>
  <Link
    replace={false}
    to="/"
  >
    Home
  </Link>
</div>
`;
